ripe/atlas/tools/commands/measure.py

```diff
@@ -480,6 +496,20 @@ def _get_source_kwargs(self):
             r["type"] = "msm"
             r["value"] = self.arguments.from_measurement
 
+        if self.arguments.include_tag or self.arguments.exclude_tag:
+            r["tags"] = {}
+            r["tags"]["include"] = self.arguments.include_tag or []
+            r["tags"]["exclude"] = self.arguments.exclude_tag or []
+        else:
+            smart_tags = conf["smart-tags"]
+            try:
+                r["tags"] = smart_tags[self.arguments.type][str(self.arguments.af)]
```

Comment:
I'm having trouble understanding what's going on here.  Is the intention to have `smart_tags` contain keys of `af`, as well as `ping`, `traceroute`, etc.?  What's the benefit here?